### ISSUE TITLE
PETSc 3.10.0 depwarning

### ISSUE THREAD

**Issue by arijitlaik** [underworldcode/underworld2#326]
This might be a bit too early to say this, but i was using petsc 3.10.0 and with mumps i get this error while running models, observed this both in OSX and linux
```
** PETSc DEPRECATION WARNING ** : the option -pc_factor_mat_solver_type is deprecated as of version 3.9 and will be removed in a future release. Please use the option -pc_factor_mat_solver_type instead. (Silence this warning with-options_suppress_deprecated_warnings)
```

**Comment by lmoresi** [comment]
Who knows - I can’t tell the difference !!! But, yes, too early until we look into this version properly.

With docker we can always choose versions and suppress warnings.

Sent from Astro<https://www.astro.ai> for iOS


On 26 Sep 2018 at 16:22, Arijit Laik <notifications@github.com> wrote:


This might be a bit too early to say this, but i was using petsc 3.10.0 and with mumps i get this error while running models, observed this both in OSX and linux

** PETSc DEPRECATION WARNING ** : the option -pc_factor_mat_solver_type is deprecated as of version 3.9 and will be removed in a future release. Please use the option -pc_factor_mat_solver_type instead. (Silence this warning with-options_suppress_deprecated_warnings)

**Comment by arijitlaik** [comment]
exactly, same thing.

**Comment by julesghub** [comment]
I'll be looking into this later on. Will keep this in mind. cheers

**Comment by arijitlaik** [comment]
Thanks :) Julian

**Comment by rbeucher** [comment]
Considering how cryptic the message is I would put a message on Petsc bitbucket repo 😁

**Comment by arijitlaik** [comment]
Really cryptic. ! ;)

**Comment by julesghub** [comment]
A petsc option name change in petsc-3.9. It affects direct solving methods and was observed if we used mumps for the inner solve. Fixed in 3ece9e3d947262e1e9351b05b89a2c9296aec4aa
